Warnings in STPAPIClient.h

Using Stripe (v7.0.2) with Cocoapods generates following warnings:

In file included from Pods/Headers/Private/Stripe/Stripe.h:9:
Pods/Headers/Private/Stripe/STPAPIClient.h:67:1: warning: no 'assign', 'retain', or 'copy' attribute is specified - 'assign' is assumed [-Wobjc-property-no-attribute]
@property (nonatomic, nonnull) NSOperationQueue *operationQueue;
^
Pods/Headers/Private/Stripe/STPAPIClient.h:67:1: warning: default property attribute 'assign' not appropriate for non-GC object [-Wobjc-property-no-attribute]
In file included from /Users/wanelo/workspace/ios-app/ios-app/wanelo-Prefix.pch:32:
In file included from Pods/Headers/Private/Stripe/Stripe.h:21:
Pods/Headers/Private/Stripe/STPPaymentCardTextField.h:250:1: warning: no 'assign', 'retain', or 'copy' attribute is specified - 'assign' is assumed [-Wobjc-property-no-attribute]
@property(nonatomic, readwrite, nonnull) STPCardParams *cardParams;
^
Pods/Headers/Private/Stripe/STPPaymentCardTextField.h:250:1: warning: default property attribute 'assign' not appropriate for non-GC object [-Wobjc-property-no-attribute]
Pods/Headers/Private/Stripe/STPPaymentCardTextField.h:252:1: warning: no 'assign', 'retain', or 'copy' attribute is specified - 'assign' is assumed [-Wobjc-property-no-attribute]
@property(nonatomic, readwrite, nullable) STPCardParams *card __attribute__((deprecated("This has been renamed to cardParams; use that instead.")));
^
Pods/Headers/Private/Stripe/STPPaymentCardTextField.h:252:1: warning: default property attribute 'assign' not appropriate for non-GC object [-Wobjc-property-no-attribute]
Pods/Headers/Private/Stripe/STPPaymentCardTextField.h:278:1: warning: no 'assign', 'retain', or 'copy' attribute is specified - 'assign' is assumed [-Wobjc-property-no-attribute]
@property(nonatomic, readwrite, nonnull) PTKCard *card;
^
Pods/Headers/Private/Stripe/STPPaymentCardTextField.h:278:1: warning: default property attribute 'assign' not appropriate for non-GC object [-Wobjc-property-no-attribute]
8 warnings generated.

--- a/Stripe/PublicHeaders/STPPaymentCardTextField.h
+++ b/Stripe/PublicHeaders/STPPaymentCardTextField.h
@@ -262,9 +262,9 @@
  *  to scan your user's credit card with a camera, you can assemble that data into an STPCardParams
  *  object and set this property to that object to prefill the fields you've collected.
  */
-@property(nonatomic, readwrite, nonnull) STPCardParams *cardParams;
+@property(nonatomic, strong, readwrite, nonnull) STPCardParams *cardParams;
 
-@property(nonatomic, readwrite, nullable) STPCardParams *card __attribute__((deprecated("This has been renamed to cardParams; use that instead.")));
+@property(nonatomic, strong, readwrite, nullable) STPCardParams *card __attribute__((deprecated("This has been renamed to cardParams; use that instead.")));
 
 - (void)commonInit;
 
@@ -292,7 +292,7 @@ __attribute__((deprecated("This protocol is provided only for backwards-compatib
 __attribute__((deprecated("This class is provided only for backwards-compatibility with PaymentKit. You shouldn't use it - use STPPaymentCardTextField instead.")))
 @interface PTKView : STPPaymentCardTextField
 @property(nonatomic, weak, nullable)id<PTKViewDelegate>delegate;
-@property(nonatomic, readwrite, nonnull) PTKCard *card;
+@property(nonatomic, strong, readwrite, nonnull) PTKCard *card;
 @end
 
 #pragma clang diagnostic pop

